pom.xml

```diff
@@ -12,12 +12,13 @@
   <artifactId>jaxen</artifactId>
   <packaging>bundle</packaging>
   <name>jaxen</name>
-  <version>1.1.6</version>
+  <version>1.1.7-SNAPSHOT</version>
```

Comment:
I'm thinking we should probably call the next release 1.2.0 since it changes the required Java version.